recipes/natural_language_processing/rag/bootc/Containerfile

```diff
@@ -0,0 +1,40 @@
+# Example: an AI powered sample application is embedded as a systemd service
+# via Podman quadlet files in /usr/share/containers/systemd
+#
+# Use build command:
+# podman build --build-arg "sshpubkey=$(cat $HOME/.ssh/mykey.pub)" -t quay.io/exampleos/rag .
+# The --build-arg "SSHPUBKEY=$(cat ~/.ssh/mykey.pub)" option inserts your
+# public key into the image, allowing root access via ssh.
+
+FROM quay.io/centos-bootc/centos-bootc:stream9
+ARG SSHPUBKEY
+
+RUN mkdir /usr/etc-system && \
```

Comment:
Yeah, I think we're going to want to do something like https://github.com/cgwalters/osbuild-cfg to avoid the copy-paste effect.  (not a blocker, just noting)

Reply:
i like it - can we just use this (everywhere?) ?